NuGet.config

```diff
@@ -4,5 +4,22 @@
   <packageSources>
     <clear />
     <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+    <add key="vstest" value="https://dotnet.myget.org/F/vstest/api/v3/index.json" />
```

Comment:
It should probably be possible to reduce the number of feeds here, but it can happen on a separate pr.